ui/lib/core/addon/components/shamir/form.hbs

```diff
@@ -0,0 +1,48 @@
+<form id="shamir" aria-label="shamir form" {{on "submit" (fn this.onSubmit this.key)}} ...attributes>
+  {{#if @errors}}
+    <div class="has-bottom-margin-m">
+      <MessageError @errors={{@errors}} />
+    </div>
+  {{/if}}
+  <div class="has-bottom-margin-m">
+    {{#if @otp}}
+      <Hds::Alert @type="inline" @color="highlight" class="has-bottom-margin-s" as |A|>
+        <A.Title>Info</A.Title>
+        <A.Description>
+          Below is the generated OTP. This will be used to encode the generated Operation Token. Make sure to save this, as
+          you will need it later to decode the Operation Token.
+        </A.Description>
+      </Hds::Alert>
+      <div class="has-background-gray-100 box has-copy-button" tabindex="-1">
+        <HoverCopyButton @copyValue={{@otp}} />
+        <h4 class="title is-7 is-marginless">
+          One Time Password (otp)
+        </h4>
+        <code class="is-word-break">{{@otp}}</code>
+      </div>
+    {{/if}}
+    {{#if (has-block)}}
+      {{yield}}
+    {{/if}}
+  </div>
+  <div class="field">
+    <label for="key" class="is-label">
+      Unseal Key Portion
+    </label>
+    <div class="control">
+      <Input class="input" @type="password" name="key" @value={{this.key}} autocomplete="off" />
+    </div>
+  </div>
+  <div class="columns is-mobile">
+    <div class="column is-narrow">
+      <button type="submit" class="button is-primary" disabled={{this.loading}} data-test-shamir-submit>
+        {{@buttonText}}
```

Comment:
It looks like this should either be the getter `this.buttonText` or `{{or @buttonText "Submit"}}`